core/src/test/java/training/taylor/timetracker/core/TrackerCoreConfigTimeEntriesTest.java

```
@@ -85,6 +85,7 @@ public void testValidReturnOfTimeEntries() {
     public void testImmutabilityOfReturnedTimeEntriesList() {
         List<TimeEntry> returnedList = trackerCoreConfig.timeEntries();
         returnedList.add(new TimeEntry());
+        throw new UnsupportedOperationException("exception thrown");
```

need to throw this exception as it is expected as mentioned in Test parameter